lib/brain.js

```diff
@@ -1883,6 +1948,14 @@ Brain.prototype.isTierCompliant = function isTierCompliant (tier) {
     case 'block':
     case 'suspend':
       return false
+    case 'custom':
+      const customInfoRequestId = tier
+      const customerData = _.find(['info_request_id', customInfoRequestId])(customer.customInfoRequestData)
+      if (!customerData) return false
+      if (customerData.approved === true) return true
+      if (customerData.approved === false) return false
+      // when approved is null then data was already added but not manually approved yet by the operator
+      return true
```

Comment:
Could this be replaced with `return !!customerData?.approved || !!customerData`?